src/container/menuitem.js

```diff
@@ -196,6 +196,21 @@ var MenuItem = Container.$extend({
     // Methods                              //
     //////////////////////////////////////////
 
+    // ====== Public methods ======
+
+    /**
+     * Destroy the widget.
+     *
+     * @method destroy
+     */
+    destroy: function () {
+        if (this.iconName && this.icon) {
+            this.icon.destroy();
```

Comment:
Is it possible to have `this.icon` not `null` but `this.iconName` at `null`?

Reply:
Nope, this.icon uses the name 
![image](https://user-images.githubusercontent.com/26410592/199693768-1162d8b3-1d62-4837-84fe-756122d0b438.png)